docs/distribution/licences.md

```diff
@@ -30,31 +31,30 @@ distributed artifacts.
 
 ### SBT
 
-We can use the plugin `sbt-license-report` to gather a list of used dependencies
-and their licences. To use it, run `enso/dumpLicenseReport` in the SBT shell.
-This will gather dependency information for all the subprojects. For each
-subproject, `license-reports` directory will be created in its `target`
-directory, containing the reports in multiple formats.
+We use a `GatherLicenses` task that uses the `sbt-license-report` and other
+sources to gather copyright information related to the used dependencies.
 
-It is important to note that the report for the root `enso` project will not
-contain dependencies of the subprojects. Instead, to list the relevant
-dependencies, you need to look at the reports for the subprojects that are
-actually built into distributable artifacts. For now these are: `runtime`,
-`runner`, `project-manager` and `launcher`.
+To configure the task, `GatherLicenses.distributions` should be set with
+sequence of distributions. Each distribution describes one component that is
+distributed separately and should include all references to all projects that
+are included as part of its distribution. Currently, we have the `launcher`
+distribution that consists of one `launcher` component and the `engine`
+distribution which includes `runtime`, `engine-runner` and `project-manager`.
 
-#### `sbt-license-report` Configuration
-
-Settings for the plugin are defined in the `licenseSettings` variable in
-[`build.sbt`](../../build.sbt). The settings have to be applied to each project
-by adding `.settings(licenseSettings)` to the project definition (defining these
-settings at the top level or in the `ThisBuild` configuration yielded no
-effects, so this workaround is required).
+The root projects that are included in `GatherLicenses.distributions` should
+have `.settings(licenseSettings)` added to configure their license information
```

Comment:
Only the root projects require it and they do have it.

Reply:
I actually changed the way I call the sbt license plugin so that this configuration is not needed anymore.
Instead we just set which configurations we want to target at the top of the `build.sbt` file, for now the setting is global (the same for all projects) as it is enough for us, but if at some point we would need more fine grained control it should not be hard to change that.